cleanlab/datalab/datalab.py

```diff
@@ -303,6 +304,71 @@ def find_issues(
                 f"\nAudit complete. {self.data_issues.issue_summary['num_issues'].sum()} issues found in the dataset."
             )
 
+    def _spurious_correlations(self, properties: Optional[List[str]] = None) -> pd.DataFrame:
+        """
+        Identify potential spurious correlations between image properties and their corresponding scores.
+
+
+        Parameters:
+        -----------
+        properties : Optional[List[str]]
+            A list of specific image properties (e.g. 'dark', 'grayscale') to be analyzed.
+            If None, all available properties from the issue summary will be considered.
+
+        Returns:
+        --------
+        A DataFrame indicating correlations for each image property.
+
+        Note
+        ----
+        This method is a wrapper around the :py:meth:`SpuriousCorrelations.calculate_correlations <cleanlab.datalab.internal.spurious_correlation.SpuriousCorrelations.calculate_correlations>` method.
+
+        It is still a work in progress and may be subject to change in future versions.
+
+        See Also
+        --------
+        cleanlab.datalab.internal.spurious_correlation.SpuriousCorrelations
+        """
+        # TODO: Update this check when support for more properties is added.
+        if self._imagelab is None:
+            raise NotImplementedError("No ImageLab instance found. Please specify properties.")
+
+        # TODO: Update this check when support for more properties is added.
+        if self._imagelab.issue_summary.empty or self.issues.empty:
+            raise ValueError("No issues found in ImageLab. Please run find_issues() first.")
+
+        # Default to all available properties from the issue summary.
+        if properties is None:
+            _issue_summary = self._imagelab.issue_summary
+            properties = _issue_summary["issue_type"].values.tolist()
+
+            # Ensure only properties present in both datalab and imagelab are considered.
+            if self._imagelab:
```

Comment:
What are we checking here? `_issue_summary` was obtained using `self._imagelab`, so is this condition required?